src/System.Management.Automation/engine/LanguagePrimitives.cs

```diff
@@ -5270,17 +5266,16 @@ internal static ConversionData FigureConversion(Type fromType, Type toType)
                     }
                 }
             }
-#if CORECLR
+
             // Assemblies in CoreCLR might not allow reflection execution on their internal types.
-            TypeInfo typeInfo = toType.GetTypeInfo();
-            if (!TypeResolver.IsPublic(typeInfo) && DotNetAdapter.DisallowPrivateReflection(typeInfo))
+            if (!TypeResolver.IsPublic(toType) && DotNetAdapter.DisallowPrivateReflection(toType))
```

Comment:
Is it a breaking change with 6.0 GA?

Reply:
Why do you think this is a breaking change?

Reply:
My understanding is that before the change we can use reflection get access to private members. If so it is _formally_ a breaking change. My request is only set the label on the PR.

Reply:
But there is no behavioral change ... The removed if/def is `#if CORECLR ... #endif`, so it has always been this way. Did you misread it to `#if !CORECLR`?

Reply:
Yes, sorry.